docs/pages-for-subheaders/rke2-hardening-guide.md

```diff
@@ -89,15 +74,15 @@ RKE2 always runs with some amount of pod security.
 
 On v1.25 and newer, [Pod Security Admissions (PSAs)](https://kubernetes.io/docs/concepts/security/pod-security-admission/) are used for pod security.
 
-Below is the minimum necessary configuration needed for hardening RKE2 to pass CIS v1.23 hardened profile `rke2-cis-1.23-hardened` available in Rancher.
+Below is the minimum necessary configuration needed for hardening RKE2 to pass CIS v1.23 hardened profile `rke2-cis-1.7-hardened` available in Rancher.
 
 ```yaml
 spec:
   defaultPodSecurityAdmissionConfigurationTemplateName: rancher-restricted
   rkeConfig:
     machineSelectorConfig:
       - config:
-          profile: cis-1.23
```

Comment:
Since the new CIS flag isn't yet implemented in RKE2, we must advise customers to use only the `cis-1.23` flag for the moment.